arbeitszeiterfassung_s4a/hooks.py

```diff
@@ -108,6 +108,9 @@
 	"Employee Checkin": {
 		"after_insert": "arbeitszeiterfassung_s4a.arbeitszeiterfassung_s4a.employee_checkin.employee_checkin.create_working_time_log",
 	},
+	"Employee Checkin": {
+		"after_save": "arbeitszeiterfassung_s4a.arbeitszeiterfassung_s4a.employee_ceckin.employee_checkin.switch_checkins_at_midnight",
+	},
```

Comment:
```suggestion
		"after_save": "arbeitszeiterfassung_s4a.arbeitszeiterfassung_s4a.employee_ceckin.employee_checkin.switch_checkins_at_midnight",
	},
```

This would be the correct syntax. But I think it doesn't make sense to run this `after_save`, so we can remove this entirely.